webalyzer/settings/base.py

```diff
@@ -154,7 +154,7 @@
 TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
 
 
-ALLIGATOR_CONN = config('ALLIGATOR_CONN', default='redis://localhost:6379/1')
+ALLIGATOR_CONN = config('ALLIGATOR_CONN', default='locmem://')
```

Comment:
I don't think that'll work. It might work for running tests. 
If you do this, you'll be running the django runserver in one python process (its own locmem) and the management command (`./manage.py run-gator`) in a separate python process.